### PR TITLE
Install slc_clean script allowing us to remove old strongloop tempora…

### PR DESCRIPTION
**Pull request by sunsongxp**
…ry files
- Add slc_clean script
- Install slc_clean script to /usr/local/bin

### CODE REVIEW

File: files/slc_clean [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
```
Comment (zbal) [inline files/slc_clean:2]: Add headers / changelog / description / purpose

File: files/slc_clean [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+             --long svc:,count:,help \
+             -- "$@")
+
+    # exit if the options have not properly been gathered
+    if [ $? != 0 ] ; then echo "Terminating..." >&2 ; exit 1 ; fi
+
+    # Note the quotes around '$OPTION': they are essential!
+    eval set -- "$OPTION"
+
+  while true ; do
```
Comment (zbal) [inline files/slc_clean]: indent - 4 spaces

File: files/slc_clean [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    echo 'Options:
+    -s, --svc               Service ID (default: 1)
+    -c, --count             Version kept (default: 2)
+    -h, --help              Show this help
+'
+    echo 'List Services:'
+    if hash slc 2>/dev/null; then
+        slc ctl ls
+    fi
+}
```
Comment (zbal) [inline files/slc_clean:31]: Add some details - such as what to do with the list being printed out.

Include an example of command

Include a `-y` | `--yes` option to auto confirm delete (see below) - default to "no"

File: files/slc_clean [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+cd $WORKDIR
+
+real_location=$(readlink current)
+
+for folder in *; do
+    if [ "$WORKDIR/$folder" != "$real_location" ]
+    then
+        if [[ $folder =~ ^[a-z0-9]{40}.[0-9]{13}$ ]]; then
+            echo $folder
+            rm -rf $folder
```
Comment (zbal) [inline files/slc_clean]: based on the `-y` | `--yes` option, automatically delete folder vs prompt user

```
if [ "$YES" != 1 ]; then
  echo -n "Gonna remove XXX [y/N] "
  read confirm
  if [ "$confirm" == 'y' -o "$confirm" == 'Y' ]; then
    rm -rf $folder
    echo "removing..."
  else
    echo "skipping..."
  fi
fi 
```

File: files/slc_clean [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    SVC=1
+fi
+
+if [ -z "$COUNT" ]; then
+    COUNT=2
+fi
+
+
+WORKDIR=/var/lib/strong-pm/svc/$SVC/work
+cd $WORKDIR
```
Comment (zbal) [inline files/slc_clean:76]: Check that $WORKDIR exist before going in - if not, print error message and exit.

File: files/slc_clean [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+fi
+cd $WORKDIR
+
+real_location=$(readlink current)
+
+for folder in *; do
+    if [ "$WORKDIR/$folder" != "$real_location" ]
+    then
+        # If folder have this patter, then try to remove this one.
+        if [[ $folder =~ ^[a-z0-9]{40}.[0-9]{13}$ ]]; then
```
Comment (zbal) [inline files/slc_clean:84]: Need to have the "else" part that would display an error message and the name of the folder that is not matching.

It should not stop the exec however.

Comment (sunsongxp) [inline files/slc_clean:84]: Done

**Pull request merged**